apps/auth0/src/post-login-add-metadata.ts

```diff
@@ -92,6 +94,9 @@ export const onExecutePostLogin = async (
 ) => {
   try {
     const [apiUrl, redirect_uri] = getApiUrls(event);
+    console.log(
```

Comment:
you might want to drop it

Reply:
Its logs on auth0 so if its comprimised its A only data we attach to the id token and B we have bigger problems